.github/workflows/canary-integration-test.yml

```
@@ -13,6 +13,11 @@ on:
     paths-ignore:
       - "Documentation/**"
       - "design/**"
+    workflow_dispatch:
+    inputs:
+      ceph-image:
+        description: 'Ceph image for creating Ceph cluster'
+        default: 'quay.io/ceph/ceph:v17'
```

```suggestion
    workflow_dispatch:
    inputs:
      ceph-image:
        description: 'Ceph image for creating Ceph cluster'
        default: 'quay.io/ceph/ceph:v17'
```

The PR is ready to, just one question @travisn 

do we want to restrict who can run the test from UI? With current changes anyone will probably be able to run the tests, but I'm not sure if there are internal setting on who can run the github action in the repo

---

There are no side effects to these tests, so for now let's not worry about a restriction

---

do you think we should remove the `default` value and use the same version that is in the `cluster-test.yaml` file or other file the test uses or does having a common value make more sense?

---

I would lean toward defining the version here by default, so it's more clear what the test is running. It's more confusing to me when we pick up the version that is in the example manifests.